sdks/python/apache_beam/transforms/external_test.py

```diff
@@ -406,6 +410,88 @@ class DataclassTransform(beam.ExternalTransform):
     return get_payload(DataclassTransform(**values))
 
 
+class JavaClassLookupPayloadBuilderTest(unittest.TestCase):
```

Comment:
Do we also need a test for setting `with_constructor_method` twice? or setting `with_constructor_method` after `with_constructor`?

Reply:
Done.